src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -327,6 +335,21 @@
         padding-right: var(--pf-c-menu-toggle--m-split-button--first-child--PaddingRight);
       }
     }
+  }
+
+  &.pf-m-input-group {
```

Comment:
I'd probably just call this typeahead. It would be good to have a modifier for the typeahead variant. Input group also is a little confusing since we have both an input group and a text input group, and I think these styles are more applicable to the use of the text input group with typeahead than, say, any time the input group is in the menu toggle.